.SysId/sysid.json

```diff
@@ -0,0 +1,11 @@
+{
+  "SysId": {
+    "Analysis Type": "General Mechanism",
+    "AnalyzerJSONLocation": "D:\\Programowanie\\FRC-2022-robot\\sysid_data20220207-141856.json",
```

Comment:
Full path

Reply:
To jest ścieżka do wygenerowanego pliku z sysid toola. Podczas uruchamiania programu można załadować plik z wygenerowanymi danymi. Za każdym razem gdy ładujemy plik ścieżka się zmienia, a dane możemy ładować nawet z pendrive.